node-src/lib/getDependentStoryFiles.test.ts

```diff
@@ -179,7 +179,10 @@ describe('getDependentStoryFiles', () => {
         id: String.raw`./src lazy recursive ^\.\/.*$`,
         reasons: [
           {
-            moduleName: './node_modules/.cache/storybook/default/dev-server/storybook-stories.js',
+            resolvedModule:
```

Comment:
If this is `rsbuild` specific, should we have a comment in the tests about that?

Reply:
This test name mentions `rspack` already. Does that cover it or should I add a note quick?

Reply:
Nope, that sounds good!  I didn't expand up to see that, my bad! 👍